app/components/reference_with_feedback_component.rb

```diff
@@ -28,10 +30,10 @@ def rows
 
   def status_row
     {
-      key: 'Feedback status',
+      key: 'Reference status',
       value: render(TagComponent,
-                    text: ApplicationReference.human_attribute_name("feedback_status.#{feedback_status}"),
-                    type: 'blue'),
+                    text: t("reference_status.#{feedback_status}"),
+                    type: color(feedback_status)),
```

Comment:
```suggestion
                    type: feedback_status == 'feedback_refused' ? 'red' : 'blue',
```

I assume Rubocop doesn't like this? 😅 

Otherwise maybe `feedback_tag_color` is less generic, but this is a nit anyway.

Reply:
Yes, Rubocop is not happy 👮 
I think `feedback_tag_color` is a sensible name, changed it!